app/Notifications/Banking/AuditResult.php

```diff
@@ -72,7 +75,7 @@ public function __construct(
      */
     public function via($notifiable)
     {
-        return ['mail', 'slack'];
+        return ['mail', 'slack', DiscordChannel::class];
```

Comment:
IIRC Discord notification channel does not handle missing token well

need to check again but having commented out or empty  `DISCORD_TOKEN` in .env might results in the following
`Unresolvable dependency resolving [Parameter #1 [ <required> $token ]] in class NotificationChannels\Discord\Discord `

if this is an issue then `DiscordChannel::class` need to be dynamically added to this list

Reply:
this would applies to all notifications

Reply:
yes you're right - thanks for spotting